oio/common/constants.py

```diff
@@ -100,6 +100,7 @@
 }
 
 CHUNK_XATTR_CONTENT_FULLPATH_PREFIX = 'oio.content.fullpath:'
+CHUNK_XATTR_CONTENT_FULLPATH_OLD_PREFIX = 'oio:'
```

Comment:
This constant already exists: https://github.com/open-io/oio-sds/blob/4.2.x/oio/blob/converter.py#L42

Reply:
Done